examples/swarm/leader-follower.py

```diff
@@ -0,0 +1,208 @@
+        yaw2.append(data['stateEstimate.yaw'])
+
+    d = math.sqrt(pow((x1[-1]-x2[-1]), 2)+pow((y1[-1]-y2[-1]), 2))
+
+
+def start_position_printing(scf):
+    log_conf1 = LogConfig(name='Position', period_in_ms=10)
+    log_conf1.add_variable('stateEstimate.x', 'float')
+    log_conf1.add_variable('stateEstimate.y', 'float')
+    log_conf1.add_variable('stateEstimate.z', 'float')
+    log_conf1.add_variable('stateEstimate.yaw', 'float')
+    scf.cf.log.add_config(log_conf1)
+    log_conf1.data_received_cb.add_callback(lambda _timestamp, data, _logconf: position_callback(scf.cf.link_uri, data))
+    log_conf1.start()
+
+
+def leader_follower(scf):
+    r_min = 0.8  # The minimum distance between the 2 drones
+    r_max = 1.0  # The maximum distance between the 2 drones
+    with MotionCommander(scf, default_height=DEFAULT_HEIGHT) as mc:
+
+        # The follower turns until it is aligned with the global coordinate system
+        while abs(yaw1[-1]) > 2:
+            if scf.__dict__['_link_uri'] == URI1:  # Follower
+                if yaw1[-1] > 0:
+                    mc.start_turn_right(36 if abs(yaw1[-1]) > 15 else 9)
+                elif yaw1[-1] < 0:
+                    mc.start_turn_left(36 if abs(yaw1[-1]) > 15 else 9)
+
+            elif scf.__dict__['_link_uri'] == URI2:  # Leader
+                mc.stop()
+            time.sleep(0.005)
+
+        time.sleep(0.5)
+
+        start_time = time.time()
+        # Define the flight time after the follower is aligned
+        end_time = time.time() + 20
+
+        while time.time() < end_time:
+
+            if scf.__dict__['_link_uri'] == URI1:  # Follower
+                if d > r_max:
+                    cmd_vel_x, cmd_vel_y = pos_to_vel(x1[-1], y1[-1], x2[-1], y2[-1], d)
+                elif d >= r_min and d <= r_max:
+                    cmd_vel_x = 0
+                    cmd_vel_y = 0
+                elif d < r_min:
+                    opp_cmd_vel_x, opp_cmd_vel_y = pos_to_vel(x1[-1], y1[-1], x2[-1], y2[-1], d)
+                    cmd_vel_x = -opp_cmd_vel_x
+                    cmd_vel_y = -opp_cmd_vel_y
+
+                mc.start_linear_motion(cmd_vel_x, cmd_vel_y, 0)
+
+            elif scf.__dict__['_link_uri'] == URI2:  # Leader
+                # Define the sequence of the leader
+                if time.time() - start_time < 3:
+                    mc.start_forward(DEFAULT_VELOCITY)
+                elif time.time() - start_time < 6:
+                    mc.start_back(DEFAULT_VELOCITY)
+                elif time.time() - start_time < 20:
+                    mc.start_circle_right(0.9, DEFAULT_VELOCITY)
+                else:
+                    mc.stop()
+
+            time.sleep(0.005)
+        mc.land()
+
+
+if __name__ == '__main__':
+    cflib.crtp.init_drivers()
+
+    factory = CachedCfFactory(rw_cache='./cache')
+    with Swarm(uris, factory=factory) as swarm:
+
+        swarm.reset_estimators()
+
+        # swarm.parallel_safe(arm)
```

Comment:
You can uncomment this so it works with the brushless out of the box. On platforms that are auto-armed sending `scf.cf.platform.send_arming_request(True)` again does not affect the behavior.